src/Servers/Kestrel/Core/src/Internal/Http3/Http3SettingType.cs

```diff
@@ -13,5 +13,18 @@ internal enum Http3SettingType : long
     /// </summary>
     MaxFieldSectionSize = 0x6,
     // https://quicwg.org/base-drafts/draft-ietf-quic-qpack.html#section-5
-    QPackBlockedStreams = 0x7
+    QPackBlockedStreams = 0x7,
+
+    /// <summary>
+    /// SETTINGS_ENABLE_WEBTRANSPORT, default is 0 (off)
+    /// https://www.ietf.org/archive/id/draft-ietf-webtrans-http3-01.html#name-http-3-settings-parameter-r
```

Comment:
Hi @Daniel-Genkin-MS-2 , just curious because I had one kesterl WebTransport implementation month ago which add this according to the draft:

// https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-h3-websockets-02#section-5
ENABLE_CONNECT_PROTOCOL = 0x8

Does it mean we actually don't have to set enable connect protocol manually and can get WebTransport worked?
Btw thanks for your great work :)

Reply:
@1someone1 your link is for WebSockets, it's a different spec. For WebTransport it says this:
>[[RFC8441](https://ietf-wg-webtrans.github.io/draft-ietf-webtrans-http3/draft-ietf-webtrans-http3.html#RFC8441)] defines an extended CONNECT method in Section 4, enabled by the SETTINGS_ENABLE_CONNECT_PROTOCOL parameter. That parameter is only defined for HTTP/2. This document does not create a new multi-purpose parameter to indicate support for extended CONNECT in HTTP/3; instead, the SETTINGS_ENABLE_WEBTRANSPORT setting implies that an endpoint supports extended CONNECT.

From: https://ietf-wg-webtrans.github.io/draft-ietf-webtrans-http3/draft-ietf-webtrans-http3.html#name-extended-connect-in-http-3